<launch>
    <!-- Number of robots used -->
    <arg name="num_of_robots" default="4"/>

    <!-- Name of the simulated or real map -->
	<arg name="map_name" default="across"/>

    <!-- Set true/false to enable/disable debug mode -->
    <arg name="debug_enabled" value="true"/>

    <!-- Control loop frequency -->
    <arg name="ctrl_loop_freq" default="10"/>

    <!-- Time before start and duration of initial velocity hold -->
    <arg name="wait_time" default="0"/>
    <arg name="start_time" default="0"/>

    <!-- Sensitivity for the joystick [0.0 - 1.0] -->
    <arg name="sensitivity" default="0.2"/>
</launch>
